### PR TITLE
Eccentric Shipyard and proper ATMs for Tinnia's

### PR DESCRIPTION
**Pull request by RealIHaveANameOfficial**
-Adds the Eccentric Shipyard to Tinnia's Rest.
-Moves several shuttles to the Eccentric Shipyard

## About the PR

Adds the Eccentric Shipyard and the Sale-Only Shipyard, with their own sprites.
Edits the expedition computer sprite to be distinct and more thematically appropriate (not an antag station, not red)
Moves some ships to the Eccentric Shipyard (Legman, Esquire, Honker, WaveShot, Bazaar, and the salvaged shuttles)
Adds several Eccentric Shipyards to Tinnia's Rest and replaces the ATMs at Tinnia's with standard, full-deposit ATMs.

## Why / Balance

More computers, more better. More unique selections of ships for different locations.

## Technical details

Will need to add taxes to the shipyards. Since I don't know what taxes to give them, I left them untaxed.
TODO: [X] Add taxes
I basically just copied the code from the blackmarket console and replaced it with the eccentric console. Works on my machine.

## Media

image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame

## Breaking changes

Shouldn't be anything. I modified the hues of some sprites to be a little more thematically appropriate.

**Changelog**

:cl:
- add: Tinnia's Rest is now offering unique shuttles for purchase!
- add: Tinnia's Rest now has proper ATMS for depositing.
- tweak: Some shuttles have been moved from the Frontier shipyard to the eccentric shipyard on Tinnia's Rest.

### CODE REVIEW

File: Content.Client/Shipyard/UI/ShipyardConsoleMenu.xaml.cs [hunk 1]
```diff
@@ -74,6 +74,8 @@ public void PopulateProducts(ShipyardConsoleUiKey uiKey)
             ShipyardConsoleUiKey.Security => "Security",
             ShipyardConsoleUiKey.BlackMarket => "BlackMarket",
             ShipyardConsoleUiKey.Expedition => "Expedition",
+            ShipyardConsoleUiKey.SaleOnly => "SaleOnly",
+            ShipyardConsoleUiKey.Specialty => "Specialty",
```
Comment (Cheackraze) [inline Content.Client/Shipyard/UI/ShipyardConsoleMenu.xaml.cs:78]: needs to be more unique and descriptive, 'specialty' is too generic

**Comment by Cheackraze** [comment]
tinnia's will need its own station and map prototype now, similar to the exped lodge or pirate's cove. In addition, it will need its station initilized properly by the game rule in the code as well.

Aside from that, I dont want to include a sale only shipyard console in that fashion. We dont really have a use for such a console right now, but if we did in the future, it would need its own custom UI. Re-using the regular shipyard console UI feels really bad.

**Review by Cheackraze — CHANGES_REQUESTED** [review]
probably some more of the ships can be put here as well, like the scrap ones for instance, as well as perhaps the rosebud and a few others.

File: Content.Client/Shipyard/UI/ShipyardConsoleMenu.xaml.cs (hunk 1 above)
Comment (RealIHaveANameOfficial) [reply Content.Client/Shipyard/UI/ShipyardConsoleMenu.xaml.cs:78]: "Eccentric" maybe? Or just simply "Tinnias" but that's boring

**Comment by RealIHaveANameOfficial** [comment]
> Aside from that, I dont want to include a sale only shipyard console in that fashion. We dont really have a use for such a console right now, but if we did in the future, it would need its own custom UI. Re-using the regular shipyard console UI feels really bad.
I'll leave it as-is at the moment since it's not being used, but I'll see if I can condense down the UI to be something a little more succinct.

File: Resources/Prototypes/_NF/Shipyard/base.yml [hunk 2]
```diff
@@ -38,6 +38,21 @@
...
+    - BaseStationJobsSpawning
+    - BaseStationRecords
+    - BaseStationShuttles
+    - BaseStationAlertLevels
+    - BaseStationSiliconLawFrontierStation
+    - BaseStationEmpImmune
+  noSpawn: true
+  components:
+    - type: Transform
+
```
Comment (Cheackraze) [inline Resources/Prototypes/_NF/Shipyard/base.yml:55]: This is unnecessary and should not have most of these properties.

File: Resources/Maps/tinnia.yml
Comment (Cheackraze) [inline Resources/Maps/tinnia.yml:1]: Tinnia's was recently updated with the contraband smuggling update, and already has full ATMs. You will need to merge master and start over your map changes.

File: Content.Server/_NF/GameRule/NfAdventureRuleSystem.cs [hunk 3]
```diff
@@ -96,7 +96,6 @@ private void OnRoundEndTextEvent(RoundEndTextAppendEvent ev)
             relayText += '\n';
             highScore.Remove(highScore.First());
         }
-        ReportRound(relayText);
```
Comment (Cheackraze) [inline Content.Server/_NF/GameRule/NfAdventureRuleSystem.cs:99]: Why is this in here? Don't do this.

File: Content.Server/_NF/GameRule/NfAdventureRuleSystem.cs
Comment (Cheackraze) [inline Content.Server/_NF/GameRule/NfAdventureRuleSystem.cs:1]: Your IDE murdered the formatting of this file, needs to be set back or atomized into a different PR.

File: Content.Server/Shipyard/Systems/ShipyardSystem.Consoles.cs [hunk 4]
```diff
@@ -326,6 +326,20 @@ public void OnSellMessage(EntityUid uid, ShipyardConsoleComponent component, Shi
             SendSellMessage(uid, deed.ShuttleOwner!, deed.ShuttleName!, component.SecurityShipyardChannel, player, true);
         }
 
+        if (ShipyardConsoleUiKey.Eccentric == (ShipyardConsoleUiKey) args.UiKey)
```
Comment (Cheackraze) [inline Content.Server/Shipyard/Systems/ShipyardSystem.Consoles.cs:329]: No taxes on civilian ship purchases please

**Review by Cheackraze — CHANGES_REQUESTED** [review]

**Comment by dvir001** [comment]
Ill be doing a rework for shipyard to clean hard coded stuff
and ill try to help with this pr after

**Comment by Everturning** [comment]
eccentric shipyard makes it much harder to get rp centered ships
please do not merge

**Comment by Cheackraze** [comment]
Feel free to come back to this, closing for now
